lib/api.common.js

```diff
@@ -594,18 +594,17 @@ api.common.sortCompareByName = (
   return 0;
 };
 
-api.common.clearCacheStartingWith = (
+api.common.clearCacheStartsWith = (
```

Comment:
OTOH, the previous name may seem to be somehow connected to time, not name prefixes.